src/sparseml/pytorch/sparsification/quantization/modifier_quantization.py

```diff
@@ -494,38 +551,105 @@ def _enable_module_qat(self, module: Module):
                     )
                 )
             module_fuse_fn(**self._model_fuse_fn_kwargs)
-        elif self._model_fuse_fn_name is None:  # default auto fn
-            self._model_fuse_fn_kwargs["inplace"] = True
-            fuse_module_conv_bn_relus(module, **self._model_fuse_fn_kwargs)
 
-        activation_qconfig_kwargs = self._get_updated_activation_qconfig_kwargs()
+        # build list of layer types that should not quantize output activations
```

Comment:
any reason this block is up here before the function call? Also given how many lines the list takes, let's try and get this moved to the helpers.py file

Reply:
Done.